### ISSUE TITLE
UserWarning: To copy construct from a tensor, ...

### ISSUE THREAD

**Issue by ZIGAIBNX** [comfyanonymous/ComfyUI#2986]
UserWarning: To copy construct from a tensor, it is recommended to use sourceTensor.clone().detach() or sourceTensor.clone().detach().requires_grad_(True), rather than torch.tensor(sourceTensor).

Help me, I've never seen this message appear, how can I solve it?
